saleor/discount/models.py

```diff
@@ -388,13 +396,40 @@ class PromotionRule(models.Model):
     reward_value = models.DecimalField(
         max_digits=settings.DEFAULT_MAX_DIGITS,
         decimal_places=settings.DEFAULT_DECIMAL_PLACES,
+        blank=True,
         null=True,
     )
 
     class Meta:
         ordering = ("name", "pk")
 
 
+class PromotionEvent(models.Model):
+    date = models.DateTimeField(auto_now_add=True, db_index=True, editable=False)
+    type = models.CharField(max_length=255, choices=PromotionEvents.CHOICES)
+    parameters = JSONField(blank=True, default=dict, encoder=CustomJsonEncoder)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        blank=True,
+        null=True,
+        related_name="promotion_events",
```

Comment:
Is there a specific requirement for a backward relation here? Do we have any plans to access promotion events differently than throughout `Promotion` objects?